### DISCUSSION TITLE
Should we release 0.68.6?

### DISCUSSION THREAD

**Discussion by cipolleschi** [reactwg/react-native-releases#45] [Patches]
### ⚠️ [Unsupported version](https://github.com/reactwg/react-native-releases#which-versions-are-currently-supported)

Current Release: [0.68.5](https://github.com/facebook/react-native/releases/tag/v0.68.5)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1. https://github.com/facebook/react-native/commit/475310dbbaec8048411edefc6cdddab330df7966

#### Local commits to backport to main

1.

**Comment by BasixKOR** [comment]
Hi, Could you consider backporting this commit to 0.68 releases? https://github.com/facebook/react-native/commit/048194849bda980eecf7bb006ca4e71e2d60ff4c

I've left a comment on the corresponding PR.

**Reply by kelset** [reply]
Hi - we are about to release 0.71 so we'll be closing off support for 0.68; we recommend you upgrade to one of the [supported versions](https://github.com/reactwg/react-native-releases#which-versions-are-currently-supported) to address your issues.

**Comment by kelset** [comment]
https://github.com/facebook/react-native/commit/be69c8b5a77ae60cced1b2af64e48b90d9955be5 (we might do it)
